lib/daru/io/io.rb

```diff
@@ -275,6 +275,17 @@ def html_table_to_dataframe(table)
           order: table[:order],
           name: table[:name]
       end
+
+      CSV::Converters[:boolean_converter] = lambda do |f, _|
```

Comment:
@zverok went by the conversation at the bottom of this for why I added it in Daru https://github.com/SciRuby/daru/issues/337